lib/Test/Nginx/Util.pm

```diff
@@ -435,7 +442,12 @@ sub add_block_preprocessor(&) {
 #our ($PrevRequest)
 our $PrevConfig;
 
-our $ServRoot   = $ENV{TEST_NGINX_SERVROOT} || File::Spec->catfile(cwd() || '.', 't/servroot');
+our $ServRoot   = $ENV{TEST_NGINX_SERVROOT} || File::Spec->rel2abs('t/servroot');
```

Comment:
can probably remove the extra whitespace between the var and the assignment operator, given that we're no longer aligning with the chunk of declarations below us